Public/Src/Tools/Execution.Analyzer/Tracing/Log.cs

```diff
@@ -5,9 +5,8 @@
 using BuildXL.Tracing;
 using BuildXL.Utilities.Instrumentation.Common;
 using BuildXL.Utilities.Tracing;
-
 #if !FEATURE_MICROSOFT_DIAGNOSTICS_TRACING

```

Comment:
Left an empty if/def.

Reply:
Will look for this in a future PR, there might be more